core/src/main/java/feast/core/model/JobStatus.java

```diff
@@ -64,4 +66,38 @@
   public static Collection<JobStatus> getTerminalState() {
     return TERMINAL_STATE;
   }
+
+  private static final Collection<JobStatus> TRANSITIONAL_STATES =
+      Collections.unmodifiableList(Arrays.asList(PENDING, ABORTING, SUSPENDING));
+
+  /**
+   * Get Transitional Job Status states. Transitionals states are assigned to jobs that
+   * transitioning to a more stable state (ie SUSPENDED, ABORTED etc.)
+   *
+   * @return Collection of transitional Job Status states.
+   */
+  public static final Collection<JobStatus> getTransitionalStates() {
+    return TRANSITIONAL_STATES;
+  }
+
+  /**
+   * Convert a Job Status to Ingestion Job Status proto
+   *
+   * @return IngestionJobStatus proto derieved from this job status
+   */
+  public IngestionJobStatus toIngestionProto() {
+    // maps job models job status to ingestion job status
+    Map<JobStatus, IngestionJobStatus> statusMap =
```

Comment:
I think this should remain non static as it allows us to write: `status.toProto();`
instead of a more verbose 
`JobStatus.toProto(status);`

> assuming the signature of the method becomes `toProto(JobStatus)`

Reply:
No, I mean the statusMap should be static, not the method.